Sources/MixedReality/Microsoft.Psi.MixedReality.UniversalWindows/SpatialAnchorHelper.cs

```diff
@@ -56,6 +56,67 @@ public SpatialAnchor TryCreateSpatialAnchor(string id, SpatialCoordinateSystem s
 
             return spatialAnchor;
         }
+        
+        /// <summary>
+        /// Creates a persisted spatial anchor at the supplied <see cref="SpatialCoordinateSystem"/> with a positional offset.
+        /// </summary>
+        /// <param name="id">The identifier of the spatial anchor.</param>
+        /// <param name="spatialCoordinateSystem">The coordinate system at which to create the spatial anchor.</param>
+        /// <param name="translation">The rigid positional offset from the coordinate system's origin to the interop frame of reference's origin.</param>
```

Comment:
>interop frame of reference [](http://example.com/codeflow?start=109&length=26)

What does "interop frame of reference" mean exactly? It might be clearer to simply remove this part of the comment.

Reply:
Ah my bad, that comment was too specific to my use case. I'm using the spatial graph interop library (https://learn.microsoft.com/en-us/uwp/api/windows.perception.spatial.preview.spatialgraphinteroppreview.createcoordinatesystemfornode?view=winrt-22621), but that isn't relevant here. Removing that part of the comment sounds good.